### ISSUE TITLE
Button: min-height for small button variant

### ISSUE THREAD

**Issue by feerglas** [lyne-design-system/lyne-components#682]
The min-height of the small button variant is 40px. We defined in our Design System, that touch-elements should have a min-height of 48px. Should we fix this?

**Comment by mcilurzo** [comment]
We have increased the minimum size of the button to 44px. This meets the criteria according to WCAG: https://www.w3.org/WAI/WCAG21/Understanding/target-size.html
